openslides_backend/action/agenda_item/delete.py

```diff
@@ -11,4 +11,4 @@ class AgendaItemDelete(DeleteAction):
     """
 
     model = AgendaItem()
-    schema = DefaultSchema(AgendaItem()).get_delete_schema()
+    schema = DefaultSchema(AgendaItem()).get_update_schema()
```

Comment:
I understand that a delete schema is the same as an update schema without properties. Nevertheless it is much more explicit if we call it `get_delete_schema` here. So we also can readd title and description of the schema.

Reply:
Seems reasonable. Will add `create/update/delete` schema methods back